### DISCUSSION TITLE
萌幻之乡 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#40] [模板讨论]
萌幻之乡 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by xlon72** [comment]
模板失效了

**Reply by wjf0214** [reply]
更新了下，再看下。

**Reply by xlon72** [reply]
错误599
Failed at 1/6 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: OpenSSL SSL_connect: Invalid argument in connection to www.hmoe11.net:443 ,
Request URL: https://www.hmoe11.net/wp-admin/admin-ajax.php?action=c8b312b2c7d65738c21c1bb6b7e7d232&4862c877a0ed054996ca7ec0d527fa10%5Btype%5D=checkUnread&be239aef638a797e4f290321a6bf6be5%5Btype%5D=getHomepagePosts

**Reply by xlon72** [reply]
我发现https://www.hmoe11.net偶尔会访问不上，先访问https://hmoe11.net，再访问https://www.hmoe11.net就可以访问

**Reply by wjf0214** [reply]
连接失败了。再试下，看看是不是网络问题，或者走个代理看看。

**Reply by xlon72** [reply]
现在有CF了

**Reply by wjf0214** [reply]
> 现在有CF了

我发你怎么过。。

**Comment by yushizui** [comment]
用不了了登录加了验证码

**Reply by wjf0214** [reply]
更新了。验证码内置的识别会识别不出来，暂时用cookie方式。

**Reply by yushizui** [reply]
辛苦了，大佬

**Comment by miaoxiaozhao** [comment]
重新开站了，之前的签到失效了，网站结构好像变了？
